share/spice/github/fatheads.json

```diff
@@ -0,0 +1,15 @@
+{
+    "jquery":       { "url": "1/jquery&ia=about" },
+    "react":        { "url": "?q=react+javascript+library&ia=about" },
+    "opencv":       { "url": "1/opencv&ia=about" },
+    "bootstrap":    { "url": "?q=bootstrap+front-end+framework&ia=about" },
+    "angular.js":   { "url": "1/angular_js&ia=about" },
+    "angular":      { "url": "1/angular_js&ia=about" },
+    "Font-Awesome":  { "url": "1/font_awesome&ia=about" },
```

Comment:
Do you know an easy way to read YAML in JS? If not lets just leave it as JSON for now to keep the PR moving.